NineChronicles.Headless/GraphTypes/States/Models/Item/CostumeType.cs

```diff
@@ -15,6 +15,10 @@ public CostumeType()
                 nameof(Costume.Equipped),
                 description: "Status of Avatar equipped."
             );
+            Field<NonNullGraphType<LongGraphType>>(
+                nameof(Costume.RequiredBlockIndex),
+                description: "Block index at the costume can use."
+            );
```

Comment:
Can you elaborate on them meaning of `RequiredBlockIndex`? I think the description needs more detail but I'm not entirely sure what it means 😅 .

Reply:
This is locking item and can't be used if the current blockchain index is lower than the `RequiredBlockIndex `given to this item.